templates/Devel.dockerfile.j2

```diff
@@ -15,11 +15,9 @@ ARG FROM_IMAGE=gpuci/rapidsai
 
 FROM ${FROM_IMAGE}:${RAPIDS_VER}-cuda${CUDA_VER}-devel-${LINUX_VER}-py${PYTHON_VER}
 
-{# Install ccache from source (use specific version in projects commit history) #}
-ARG CCACHE_VERSION=3.7.11
-RUN curl -s -L https://github.com/ccache/ccache/releases/download/v${CCACHE_VERSION}/ccache-${CCACHE_VERSION}.tar.gz -o ccache-${CCACHE_VERSION}.tar.gz \
- && tar -xvzf ccache-${CCACHE_VERSION}.tar.gz && cd ccache-${CCACHE_VERSION} \
- && ./configure && make install -j16 && cd - && rm -rf ./ccache-${CCACHE_VERSION} ./ccache-${CCACHE_VERSION}.tar.gz
+{# Install ccache from conda package #}
+RUN source activate rapids
+RUN conda install -c gpuci gpuci-ccache
```

Comment:
```suggestion
RUN source activate rapids && \
    conda install -c gpuci gpuci-ccache
```

I think these two lines need to be consolidated in order for the `rapids` environment to be utilized during the `conda install`. Feel free to double-check this though.

Reply:
Good catch, ccache should be in the base environment anyways so I just removed the `source activate rapids` part.